odoo/service/db.py

```diff
@@ -275,13 +275,13 @@ def chunks(d, n=8192):
         os.unlink(data_file.name)
     return True
 
-@check_db_management_enabled
```

Comment:
Why did you remove this?

Reply:
Because it seemed useless initially and I hadn't overridden `list_db`. Good spot.

If I remove any it's probably the one on `exp_restore` which is redundant as the http endpoint calls restore_db directly. Same for `dump_db`.

Reply:
I'm not sure it is useful but better safe than sorry :) With the config hack it should work fine.